web/uk-base-world/scripts/start.sh

```diff
@@ -98,30 +105,35 @@ then
 
     # Copy in the stack uploader config(s)
     cp "./inputs/config/dukes_2023.json" "$UPLOAD_CONFIG/"
+    cp -r ./inputs/config/pylons-and-veg/* "$UPLOAD_CONFIG/"
 
     # Copy in the data for upload
-    UPLOAD_DATA="$ROOT/Deploy/stacks/dynamic/stack-data-uploader/inputs/data"
+    #UPLOAD_DATA="$ROOT/Deploy/stacks/dynamic/stack-data-uploader/inputs/data"
```

Comment:
Commenting out this line breaks the subsequent copy calls.

Reply:
Forgot to comment this when I'd done so for debugging, thanks